src/cs-graphics/HDRBuffer.hpp

```diff
@@ -44,7 +44,7 @@ class CS_GRAPHICS_EXPORT HDRBuffer {
   HDRBuffer(HDRBuffer&& other)      = delete;
 
   HDRBuffer& operator=(HDRBuffer const& other) = delete;
-  HDRBuffer& operator=(HDRBuffer&& other) = delete;
+  HDRBuffer& operator=(HDRBuffer&& other)      = delete;
```

Comment:
To make the clang-format version of the GitHub actions runner happy :smile: 
```suggestion
  HDRBuffer& operator=(HDRBuffer&& other) = delete;
```